ensime_shared/config.py

```diff
@@ -55,7 +56,8 @@
     "split_window": "split {}",
     "vert_split_window": "vsplit {}",
     "new_vertical_window": "{}vnew {}",
-    "new_vertical_scratch": "{}vnew {} | setlocal nobuflisted buftype=nofile bufhidden=wipe noswapfile",
+    "new_vertical_scratch":
+        "{}vnew {} | setlocal nobuflisted buftype=nofile bufhidden=wipe noswapfile",
```

Comment:
This is somewhat out of scope, but since it relates to style it's probably appropriate to bring it up now: I don't know how I feel yet about the `commands` dictionary in general, for `feedback` I think it's sensible and the fact that it's been used thus far would make it pretty easy to add localization to our plugin, a beneficial side effect.

For commands, I wonder if it would be better to encapsulate them as methods on some kind of `Editor` object instead. That would probably help a good deal with testability, be more readable than `self.vim.command(commands['foo_bar'].format("whatever"))` sprinkled all over the code base, and would provide nice go-to-def and autocompletion with argument hints via jedi-vim 😄 

Whether or not we try that, or in the interim, I strongly wish for all of these format strings to require keyword parameters, e.g.

```python
commands = {
    # ...
    "split_window": "split {filename}",
    # ...
}

# call site
self.vim.command(commands['split_window'].format(filename="foo.scala"))
```

Short of actual object methods, that at least makes things a bit less opaque as you're reading code IMO, and less error-prone.